open-company/open-company-web
# Code blocks

Card: https://trello.com/c/6MRWcY4W

Related PR:
- [Interaction](https://github.com/open-company/open-company-interaction/pull/43)
- [Lib](https://github.com/open-company/open-company-lib/pull/80)

To test:
- try adding a code block with multiple lines using the picker button
- try adding a code block with multiple lines using the ``` starter
- try adding an inline code using the format button
- try adding an inline code using the ` starter/closer
- make sure you can't format text inside code blocks (both type)
- copy some text that include a `PRE` element from an HTML page and paste it in our editor
- do you see the PRE elements transformed into code?
- do you NOT see text formatted with bold or italic inside etc?
- repeat the copy/paste with a `CODE` element
- test block codes in comments
- make sure digest looks good if there is a post with code blocks
- make sure Slack/email shares looks good if there is a post with code blocks
- make sure notifications (in-app, email, slack) looks good if there are code blocks
- make sure code blocks look good in FoC
- make sure code blocks look good in dark mode
